cranelift/codegen/src/isa/x86/enc_tables.rs

```diff
@@ -1376,6 +1378,8 @@ fn convert_ushr(
     let mut pos = FuncCursor::new(func).at_inst(inst);
     pos.use_srcloc(inst);
 
+    let isa_flags = isa_settings::Flags::new(&isa.flags(), isa_settings::builder());
```

Comment:
Just a drive-by comment: I'm not too sure this will work. To get the ISA flags I had to add some extra machinery (see https://github.com/bytecodealliance/wasmtime/pull/1759/commits/849d71bdb7bbcc3c6f6975dfb2224e75185dab06)

Reply:
And it's use: https://github.com/bytecodealliance/wasmtime/pull/1759/commits/6ab817eae953cfb4fd76b62d94bf89408e289b94#diff-f5aa2e18f8d875720c00f30b86cb69d4R1523

Reply:
Yeah, you're right, it doesn't work. Which PR is that machinery added in?

Reply:
#1759, which should be ready to go; @bnjbvr, here's another user for the ISA-specific flag machinery I added.

Reply:
(Oh, sorry, @whitequark, I see you already figured that out below!)